src/AppBundle/Controller/BuilderController.php

```diff
@@ -549,7 +549,8 @@ public function saveAction (Request $request)
 		$em = $this->getDoctrine()->getManager();
 
 		$user = $this->getUser();
-		if (count($user->getDecks()) > $user->getMaxNbDecks())
+		$decks = $em->getRepository('AppBundle:Deck')->findBy(["user"=> $user->getId(), "nextDeck" => null], array("dateUpdate" => "DESC"));
```

Comment:
Main change is here @Kamalisk, I made it match what is being sent to the twig down below, in case you want to add it separately, apologies about my editor trimming trailing whitespace.